projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -441,21 +440,13 @@
 	.wp-block-jetpack-field-option-checkbox,
 	.wp-block-jetpack-field-option-radio {
 		display: flex;
-		align-items: baseline; /* Align input with first label line */
+		align-items: center;
```

Comment:
Apologies for the quick drive-by, but I think one of the reasons why `align-items: center` wasn't used previously is that it's possible for a checkbox item to run across multiple lines. In that case, the previous idea was that the checkbox would be aligned with the baseline of the first row of text. I.e. here's how it looks prior to the inner blocks migration:

<img width="1123" alt="image" src="https://github.com/user-attachments/assets/40a02645-b251-461a-91f3-7b5211183e73" />

It's a bit more obvious when using narrower width values for the field.

Whereas I think `align-items: center` might wind up looking like this:

<img width="225" alt="image" src="https://github.com/user-attachments/assets/c7c8449f-767d-426f-b084-93e9e3bcb87e" />

Reply:
For a bit more history on this styling, here's where it was fixed previously in https://github.com/Automattic/jetpack/pull/34429, which followed on from https://github.com/Automattic/jetpack/pull/34272.

Reply:
Oh, thanks for that. Okay, I'll play around with baseline/start or something.

Reply:
Also I tried a fix in https://github.com/Automattic/jetpack/pull/42850, but decided to pause it for the inner blocks refactor

Reply:
Hmm, I see now. It's hard to get pixel perfect here. I see why the `translateY` was used. 

The other option is to reset the line height to remove the white space around the label text, but that makes wrapped text look squished.

I've fudged it for now.

Reply:
> Also I tried a fix in https://github.com/Automattic/jetpack/pull/42850, but decided to pause it for the inner blocks refactor, the baseline alignment was also mentioned there and confirmed by design.

Thanks @talldan 

Your fix looks better. I can remove mine here.

Reply:
Removed!

Reply:
> Your fix looks better. I can remove mine here.

@ramonjd That PR is based on trunk and a little older, not sure if it will still work against the inner blocks refactor.

Reply:
Oh! No worries, happy to reinstate and take back everything tomorrow. Thanks for the tip off

Reply:
Recommitted in [35562fa](https://github.com/Automattic/jetpack/pull/43531/commits/35562fa35dbe2e5ae0b3605fa32f3f6a4c0fa001)